rnd/autogpt_server/autogpt_server/schema.prisma

```diff
@@ -0,0 +1,67 @@
+datasource db {
+  provider = "sqlite"
+  url      = "file:./database.db"
+}
+
+generator client {
+  provider  = "prisma-client-py"
+  interface = "asyncio"
+}
+
+// This model describe a single node in the Agent Graph/Flow (Multi Agent System).
```

Comment:
would we want a graph db here?

Reply:
@aarushik93 do you know anything about this

Reply:
Graph db is all about performance optimization, having our system bottlenecked by an LLM, also running a JVM on our system might be too resource hogging ? :D

Reply:
My light thoughts are we will want it one day, may as well add it today so the migration is easier?

Reply:
Any idea on what benefit we will get from it?

Reply:
https://discord.com/channels/1126875755960336515/1247160510819074048/1247765794750861433